apps/onlyoffice.sh

```diff
@@ -74,23 +139,6 @@ check_open_port 443 "$SUBDOMAIN"
 # Install Docker
 install_docker
 
-# Check if OnlyOffice or Collabora is previously installed
-# If yes, then stop and prune the docker container
-docker_prune_this 'onlyoffice/documentserver'
-docker_prune_this 'collabora/code'
-
-# Disable RichDocuments (Collabora App) if activated
-if [ -d "$NC_APPS_PATH"/richdocuments ]
-then
-    occ_command app:remove richdocuments
-fi
-
-# Disable OnlyOffice (Collabora App) if activated
-if [ -d "$NC_APPS_PATH"/onlyoffice ]
-then
-    occ_command app:remove onlyoffice
-fi
-
 # Install Onlyoffice docker
 docker pull onlyoffice/documentserver:latest
```

Comment:
You can remove OnlyOffice from this PR as we will do that separately.

Reply:
Okay 👍